package.json

```
@@ -94,10 +95,13 @@
     "coverage": "c8 report --check-coverage",
     "prepare": "npm run compile",
     "pretest": "npm run compile -- -p tsconfig.test.json && cross-env NODE_ENV=test npm run build:cjs",
-    "compile": "rimraf build && tsc",
+    "compile": "rimraf build && tsc -b tsconfig.json",
+    "compile:watch": "npm run compile -- --watch",
```

Is `--watch` built into newer versions of npm?

---

1) It does involve some copy-paste repetition, but I prefer the pattern you used for `build:cjs:watch` where you copied the command so you could add the watch.
2) the `dev` script does a compile, then runs compile-watch, which does a compile, which includes a rimraf which runs in parallel to `build:cjs:watch`. I'll make a comment there too...

So if the combination of comments makes sense, I suggest

```suggestion
    "compile:watch": "tsc --watch -p tsconfig.json",
```

---

I realised I am comfortable passing command-line flags into `npm run-script` from the command-line, just don't normally do it between run-scripts, so weakening my suggestion above. Just an alternative approach!